packages/tempus/src/Tempus_StepperExplicitRK_impl.hpp

```diff
@@ -321,8 +302,14 @@ void StepperExplicitRK<Scalar>::takeStep(
 
     // Compute stage solutions
     for (int i=0; i < numStages; ++i) {
-      if (!Teuchos::is_null(stepperExplicitRKObserver_))
-        stepperExplicitRKObserver_->observeBeginStage(solutionHistory, *this);
+        this->stepperObserver_->observeBeginStage(solutionHistory, *this);
+
+        // ???: is it a good idea to leave this (no-op) here?
+        this->stepperObserver_
+            ->observeBeforeImplicitExplicitly(solutionHistory, *this);
+
+        // ???: is it a good idea to leave this (no-op) here?
+        this->stepperObserver_->observeBeforeSolve(solutionHistory, *this);
```

Comment:
Three here.  I think it might be confusing to have back-to-back observer calls.  Is there a way to avoid this?